src/AST/Entities.dfy

```diff
@@ -80,6 +82,10 @@ module {:options "-functionSyntax:4"} Bootstrap.AST.Entities
     static function EMPTY(): Location {
       Location("<none>", 0, 0)
     }
+
+    function ToString(): string {
+      file + "(" + NumUtils.IntToString(line) + "," + NumUtils.IntToString(column) + ")"
```

Comment:
We have `Utils.Lib.Str .of_int` for this, no externs needed :)

Reply:
Oh, nice!